### PR TITLE
Adding high performance computers installation guide

### PR DESCRIPTION
**Pull request by karanphil**
Here is the installation guide for Beluga!

@arnaudbore can you just confirm that the `module load` are up-to-date?

Should be good to review now!

### CODE REVIEW

File: source/arriving/setup_computer.rst [hunk 1]
```diff
@@ -157,12 +157,43 @@ Nextflow
...
+
+Connect to Beluga via ssh with :bash:`ssh USER@beluga.computecanada.ca`.
+
+On your first visit, you will probably want to edit your .bashrc with your preferences. Since VSCode will not be available, you will have to use an editor built in the terminal like Nano (:bash:`nano ~/.bashrc`) or Vim (:bash:`vim ~/.bashrc`). Please refer to the :ref:`ref_linux` if you don't know these tools.
+
+Everytime you log in Beluga, you will need to load the modules necessary for your needs (scilpy, tractoflow, etc). Here are the modules currently needed for running Nextflow pipelines:
+
+.. code-block:: bash
+
+    module load StdEnv/2020
```
Comment (CHrlS98) [inline source/arriving/setup_computer.rst:173]: Based on https://docs.alliancecan.ca/wiki/Standard_software_environments#StdEnv/2020 `StdEnv/2020` is still actively supported but not sure... @arnaudbore ?

File: source/arriving/setup_computer.rst [hunk 2]
```diff
@@ -157,12 +157,43 @@ Nextflow
...
+    module load nextflow/21.10.3
+    module load apptainer/1.1.8
+
+However, if you want to install scilpy, open a new session and follow these steps:
+
+.. code-block:: bash
+
+    module load StdEnv/2023 python/3.10 vtk
+    virtualenv --no-download --clear ~/Envs/scilpy && source ~/Envs/scilpy/bin/activate
```
Comment (CHrlS98) [inline source/arriving/setup_computer.rst:182]: Will this create the `~/Envs` directory if it does not already exist?

File: source/arriving/setup_computer.rst [hunk 3]
```diff
@@ -157,12 +157,43 @@ Nextflow
...
+    python -c 'import scilpy'
+
+In the future, you will need to follow these steps to work with scilpy after opening a new session:
+
+.. code-block:: bash
+
+    module load StdEnv/2023 python/3.10 vtk
+    source ~/Envs/scilpy/bin/activate
+
+Note that it is currently not possible to work with both scilpy and Nextflow on the same session, as they require different module versions.
```
Comment (CHrlS98) [inline source/arriving/setup_computer.rst:194]: I would add: If you need both these packages at the same time, you might want to consider using containers [ref containers].

**Review by CHrlS98 — COMMENTED** [review]
Here are my comments.

File: source/arriving/setup_computer.rst (hunk 1 above)
Comment (karanphil) [reply source/arriving/setup_computer.rst:173]: Yes it is!

File: source/arriving/setup_computer.rst (hunk 2 above)
Comment (karanphil) [reply source/arriving/setup_computer.rst:182]: Yes!

**Review by CHrlS98 — APPROVED** [review]

**Pull request merged**